mne/viz/topomap.py

```diff
@@ -761,7 +761,7 @@ def _plot_ica_topomap(ica, idx=0, ch_type=None, res=64, layout=None,
 
     if merge_grads:
         from ..channels.layout import _merge_grad_data
-        data = _merge_grad_data(data)
+        data = _merge_grad_data(data.ravel())
```

Comment:
_merge_grad_data docstring says it expects 

data : array, shape = (n_channels, n_times)

so I guess ravel() is bad or docstring is wrong